aiida/cmdline/commands/cmd_database.py

```diff
@@ -8,95 +8,61 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """`verdi database` commands."""
+# pylint: disable=unused-argument
```

Comment:
Wouldn't disable this on the global scope.

Reply:
Normally I would agree, but I think it makes sense to make it general here. The reason I need to use this is that all the functions in this module will just be warning without actually doing anything with their arguments (except for `migrate` but since it forwards the call in a way that does not explicitly pass the arguments I am unsure pylint will recognize it 🤔), but I still want the arguments to be there so that users passing them get the deprecation warning and not some wrong argument error. This logic applies to all the functions in the module in general.

Does this make sense or do you still prefer if I put this on each of the commands?

Reply:
Bah, unless the deprecation decorator changes this behavior and warns irrespective of what extra parameters I pass?

Reply:
Fair enough, can leave it as is since there is no functional code anymore and we don't expect to add anything in it as it will be removed entirely at some point